src/common/Throttle.cc

```diff
@@ -368,3 +368,61 @@ void OrderedThrottle::complete_pending_ops() {
     ++m_complete_tid;
   }
 }
+
+uint32_t DynamicThrottle::calc_wait_time(uint32_t percentage_empty, 
+                  double &wait_time, bool return_start_delay_if_none) {
+
+  bool need_to_delay = false;
+  if (percentage_empty > 100) {
+    return 1;
```

Comment:
If user of this interface provides the percentage_empty param value > 100 , I am bailing out.